### ISSUE TITLE
[Bug] in gameTouch input

### ISSUE THREAD

**Issue by jinkazama82it** [Ryochan7/DS4Windows#570]
In latest update (for now I tested it only on monster hunter world) the touch input stop to work during the game, at the moment I returned to 1.68 version for solve the issue and play mhw.

**Comment by Ryochan7** [comment]
Can you share your profile? All touch input bindings are working for me. Also, can you confirm that touch input bindings do not work on the desktop as opposed to just not working in game?

**Comment by jinkazama82it** [comment]
I make some other and more in deep check, the issue is the share and option button.
more precisely i seen this behavior the xbox controller back button input work only if bound to share button of ds4, start input work with other button (included touch input) only if bound even with option button of ds4, no other option are accepted all this in MHW.
in late night i make some test with different games for verify if is only an issue with MHW or could be extended to all games.

**Comment by Ryusennin** [comment]
No problem here (Win7-64, DS4W 1.6.9) with the Touchpad assigned to Share and Options. Tested with a few games on Steam (Hollow Knight) and Epic Store (Axiom Verge).

**Comment by Ryochan7** [comment]
I have now confirmed a problem. An easy way to produce the problem is to map Cross to B and Circle to A. Pressing Cross will output B but pressing Circle will not output A. It is a state field resetting problem. One binding will end up resetting the results from another binding.

**Comment by Ryochan7** [comment]
I am still testing out the most recent change but it should take care of the problem. Remapping XInput controls was a two step process with previous releases but I tried to change that with version 1.6.9. I tested some remapping routines and the results were fine with what I tested. A modified two step routine has been implemented in the latest commit and it should be slightly more efficient than what was used before.
